DiceKitTests/MultiplicationExpression_Tests.swift

```diff
@@ -131,6 +136,22 @@ extension MultiplicationExpression_Tests {
         expect(result.multiplicandResults) == expectedMultiplicandResults
     }
     
+    func test_probabilityMass_shouldReturnCorrect() {
+        property("probability mass") <- forAll {
+            (a: Int, b: Int) in
+            
+            let a = c(a)
+            let b = c(b)
+            
+            let expectedProbMass = a.probabilityMass.product(b.probabilityMass)
+            let expression = MultiplicationExpression(a, b)
+            
+            let probMass = expression.probabilityMass
+            
+            return probMass == expectedProbMass
+        }
+    }
+    
 }
 
 func ==(lhs: MultiplicationExpression_Tests.MockExpression, rhs: MultiplicationExpression_Tests.MockExpression) -> Bool {
```

Comment:
Not part of your diff, but we should add a space between the operator and argument list.